packages/compiler/src/render3/view/i18n/get_msg_utils.ts

```diff
@@ -11,7 +11,7 @@ import * as o from '../../../output/output_ast';
 
 import {serializeIcuNode} from './icu_serializer';
 import {i18nMetaToJSDoc} from './meta';
-import {formatI18nPlaceholderName} from './util';
+import {formatI18nPlaceholderName, i18nFormatPlaceholderNames} from './util';
```

Comment:
Those function names are confusing/inconsistent :(
It's pre-existing, so feel free to ignore (I can create a followup PR to rename them).
```suggestion
import {formatI18nPlaceholderName, formatI18nPlaceholderNamesInMap} from './util';
```

Reply:
Renamed to `formatI18nPlaceholderNamesInMap`.